src/Sylius/Behat/Context/Ui/Shop/AddressBookContext.php

```diff
@@ -105,6 +105,17 @@ public function iEditAddressOf($fullName)
     }
 
     /**
+     * @Given my default address is of :fullName
```

Comment:
`Given` should be placed in `Setup/*Context` and should be done programmatically.